### PR TITLE
feat: Updated result styles

### PR DESCRIPTION
**Pull request by jmoranin98**
#2 
screenshot

### CODE REVIEW

File: src/styles/Results.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@import url('https://fonts.googleapis.com/css?family=Lato|Roboto+Mono&display=swap');
```
Comment (lemieszek) [inline src/styles/Results.scss]: import fonts globally

File: src/styles/Results.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.result {
```
Comment (lemieszek) [inline src/styles/Results.scss]: yet another file with `.result` class :(

File: src/components/core/Chip.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '../../styles/Chip.scss'
+import { getContrastTextColor } from '../../utils/color';
+
+const Chip = ({ text, color }) => {
```
Comment (lemieszek) [inline src/components/core/Chip.jsx:5]: 👏  great!

**Review by lemieszek — CHANGES_REQUESTED** [review]
there is other file with a `result` class. Do not make spaghetti

**Comment by trybick** [comment]
@jmoranin98 Thank you! Looks amazing.

@d-szymanski Thanks for the tips

🚀

**Pull request merged**